src/Dijkstra/AST/RWS.agda

```diff
@@ -94,24 +94,24 @@ module RWSBase where
 
   open ASTPredTrans
   RWSPT : ASTPredTrans RWSOps RWSTypes
-  returnPT RWSPT x P (ev , st) =
-    P (x , st , [])
-  bindPT   RWSPT f (ev , st) P (x , st' , outs) =
-    ∀ r → r ≡ x → f r (RWSbindPost outs P) (ev , st')
-  opPT     RWSPT (RWSgets g)          f P (ev , st) =
-    P (g st , st , [])
-  opPT     RWSPT (RWSputs p refl)     f P (ev , st) =
-    P (unit , p st , [])
-  opPT     RWSPT (RWSask refl)        f P (ev , st) =
-    P (ev , st , [])
-  opPT     RWSPT (RWSlocal l)         f P (ev , st) =
-    ∀ ev' → ev' ≡ l ev → f (lift unit) P (ev' , st)
-  opPT     RWSPT (RWStell out refl)   f P (ev , st) =
-    P (unit , st , out)
-  opPT     RWSPT (RWSlisten{A'} refl) f P (ev , st) =
-    f (lift unit) (RWSlistenPost P) (ev , st)
-  opPT     RWSPT{A} RWSpass           f P (ev , st) =
-    f (lift unit) (RWSpassPost P) (ev , st)
+  returnPT RWSPT x P (e , s) =
+    P (x , s , [])
```

Comment:
align with lines 102, 104, 106, ...

Reply:
You are the king of spotting inconsistent alignment!  DONE, thanks.